src/templates/article.js

```diff
@@ -89,14 +89,20 @@ const ArticleSeries = ({ allSeriesForArticle, slugTitleMapping, title }) => {
         return mappedSeries;
     };
 
-    return Object.keys(allSeriesForArticle).map(series => (
-        <>
-            <Series name={series} currentStep={title}>
-                {getMappedSeries(allSeriesForArticle[series])}
-            </Series>
-            <br />
-        </>
-    ));
+    return Object.keys(allSeriesForArticle).map(series => {
+        const seriesItems = getMappedSeries(allSeriesForArticle[series]);
+        const activeSeriesIndex = seriesItems.findIndex(
```

Comment:
I wonder if we can reduce the number of loops we have here now that it's 3. What do you think?

Reply:
Oh yeah great point, we can definitely move this check into the `getMappedSeries` helper, nice find!